front/src/locales/fr.json

```diff
@@ -1085,6 +1084,17 @@
       "right": "Droite",
       "center": "Centré",
       "justify": "Justifié"
+    },
+     "roles": {
+      "OWNER": "Propriétaire",
+      "ADMIN": "Admin",
```

Comment:
Do you think "Administrateur" could be a better translation ?
Idem for profiles ?

Reply:
I think we are all used to seeing/using Admin.

Reply:
Mmm... well, I do not agree, sometimes it's translated as "Administrateur" entirely (eg, on Linux Mint OS). 
But, ok, we'll see later if people don't understand.